File: java/Processor.java
package DPBO.TP.TP3.java;

public class Processor {
    private String NameProcessor;
    private int PriceProcessor;

    public Processor() {
    }

    public Processor(String Name, int Price) {
        this.NameProcessor = Name;
        this.PriceProcessor = Price;
    }

    public void setNameProcessor(String Name) {
        this.NameProcessor = Name;
    }

    public String getNameProcessor() {
        return this.NameProcessor;
    }

    public void setPriceProcessor(int Price) {
        this.PriceProcessor = Price;
    }

    public int getPriceProcessor() {
        return this.PriceProcessor;
    }

    public void print() {
        System.out.println("Processor :");
        System.out.println();
        System.out.println("Name :" + this.getNameProcessor());
        System.out.println("Price (Rp) :" + this.getPriceProcessor());
    }

}

File: php/Processor.php
<?php

class Processor{
    private $NameProcessor;
    private $PriceProcessor;

    function __construct($Name, $Price)
    {
        $this->NameProcessor = $Name;
        $this->PriceProcessor = $Price;
    }

    function setNameProcessor($Name){
        $this->NameProcessor = $Name;
    }

    function getNameProcessor(){
        return $this->NameProcessor;
    }

    function setPriceProcessor($Price){
        $this->PriceProcessor = $Price;
    }

    function getPriceProcessor(){
        return $this->PriceProcessor;
    }

    function print(){
        echo "Processor :" . "<br/>";
        echo "Name :" . $this->getNameProcessor(). "<br/>";
        echo "Price (Rp) :" . $this->getPriceProcessor(). "<br/>";
    }
}
?>